scripts/gui.lua

```diff
@@ -1,11 +1,12 @@
 local GUI = {
-    allTrackers = { "area", "base", "player", "rocket" },
-    allTrackersLabels = { { "tracker-area" }, { "tracker-base" }, { "tracker-player" }, { "tracker-rocket" } },
+    allTrackers = { "area", "base", "player", "rocket", "cityblock" },
+    allTrackersLabels = { { "tracker-area" }, { "tracker-base" }, { "tracker-player" }, { "tracker-rocket" }, { "tracker-cityBlock" } },
     allTrackersLabelsMap = {
         area = { "tracker-area" },
         base = { "tracker-base" },
         player = { "tracker-player" },
-        rocket = { "tracker-rocket" }
+        rocket = { "tracker-rocket" },
+        cityBlock = { "tracker-cityBlock" },
```

Comment:
Can you order alphabetically? It looks nicer in the GUI (dropdown)

At least in English...